tests/func/experiments/test_experiments.py

```diff
@@ -34,6 +39,13 @@ def test_new_simple(tmp_dir, scm, dvc, exp_stage, mocker):
     ).read_text() == "foo: 2"
 
 
+@pytest.mark.skipif(os.name == "nt", reason="Not supported for Windows.")
```

Comment:
NTFS also doesn't have +x, right? Just clarifying.

We have some `not supported on windows` tests that are really about ntfs and not windows itself. Just important for us to keep that in mind :slightly_smiling_face:

Reply:
in Windows Python's `os.chmod` ignores everything except for the user read/write bits, regardless of filesystem, so for this test I think it's not NTFS specific.

As far as using NTFS outside of Windows, user's would probably run into issues w/experiments and properly preserving file mode unless they use git with `core.filemode false`, but that's not really a DVC specific issue